src/components/atoms/Toggler/Toggler.tsx

```diff
@@ -1,24 +1,8 @@
-import React, { ChangeEventHandler } from "react";
+import React from "react";
 
 import { Checkbox, CheckboxProps } from "components/atoms/Checkbox";
 
-export interface TogglerProps extends CheckboxProps {
-  defaultToggled: boolean;
-  onToggle: ChangeEventHandler<HTMLInputElement>;
-  containerClassName?: string;
-}
```

Comment:
To be restored + fixed up.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1775/commits/499e6b522507d0bbfdc29e110e0fddd441de1af4